plugins/automelt.cpp

```diff
@@ -265,27 +268,40 @@ static bool isStockpile(df::building * building) {
     return building->getType() == df::building_type::Stockpile;
 }
 
+struct BadFlagsCanMelt {
+    uint32_t whole;
+
+    BadFlagsCanMelt() {
```

Comment:
should this have artifact, spider_web, and owned too?

and for both of them, maybe take away rotten? is there anything meltable that can rot? can it still be melted even if it's rotten?

Reply:
That is a good question. I don't honestly know if there are rotten meltable items, perhaps some metal-based forgotten beast bits could count? I don't know if they have a `craft_material_class` of Metal or not.